api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -67,6 +68,9 @@ public void runBeforeEachTest() {
 			encounterService = Context.getEncounterService();
 			cs = Context.getConceptService();
 		}
+
+        // taking dao from the spring application context object
+        pws = (ProgramWorkflowService) applicationContext.getBean("programWorkflowDAO");
```

Comment:
You should be testing ProgramWorkflowService and not the DAO

Reply:
@wluyima  I will correct this one. Thanks